### ISSUE TITLE
WCAG Compliance: 1.4.3 Contrast

### ISSUE THREAD

**Issue by owenslo** [RunestoneInteractive/rs#98]

**Comment by oscarlevin** [comment]
Bumping this!  I am getting contrast errors on the green buttons for runestone components.  It should be straight forward to darken the green to meet the 4.5:1 contrast requirements.

**Comment by bnmnetp** [comment]
Maybe, maybe not.... We have used bootstrap for years and years to manage the buttons and colors.  I'm surprised that bootstrap is not compliant.

**Comment by oscarlevin** [comment]
So this is interesting.  Looking in DMOI4 on Runestone, buttons for multiple choice and free response and others have background color #5cb85c, set by `bootstrap.css`.  However, in a non-pretext book, I see a green button with background color #427e44, set by some `accessibility.css?v=50109A2C`.  

The `accessibility.css` color has a ratio of 4.88:1, while the `bootstrap.css` color has a ratio of 2.48:1.  

I don't understand where these colors are coming from; I poked around a bit but couldn't figure out where to start a PR.

**Comment by bnmnetp** [comment]
I added `@import (less) url("../../accessibility/css/accessibility.css");` to `activecode.less`. Then I fixed the values for the default button in accessibility.css

### Before

<img width="994" alt="Image" src="https://github.com/user-attachments/assets/2b83a17c-a5c0-40cc-82a3-3e1f0082b7d2" />

### After

<img width="1033" alt="Image" src="https://github.com/user-attachments/assets/2ba7b584-f55b-46f7-9407-e722ef869ff3" />

**Comment by bnmnetp** [comment]
I moved the import to ptxrs-bootstrap.less so it applies to all of the components.

I'll commit this as part of a number of other a11y updates I've been working on.

**Comment by ascholerChemeketa** [comment]
I did a minimal test of it. Looks good.

I'm not surprised that Bootstrap 3.4.1 has accessibility issues. It dates to ~2013. Accessibility was a thing back then, but not in the same way.

It seems like it would be good to either upgrade the boostrap version or to phase out bootstrap for the components. Phasing out would be easier to do component by component in a gradual way. Upgrading bootstrap would be nice in terms of keeping the UI of components at least semi unified.
